Hearthstone Deck Tracker/Controls/Overlay/Battlegrounds/HeroPicking/BattlegroundsHeroPicking.xaml.cs

```diff
@@ -10,6 +11,16 @@ public BattlegroundsHeroPicking()
 			InitializeComponent();
 		}
 
+		private void OverlayVisibilityToggle_MouseUp(object sender, System.Windows.Input.MouseButtonEventArgs e)
+		{
+			var viewModel = (BattlegroundsHeroPickingViewModel)DataContext;
```

Comment:
You should make this `if(DataContext is BattlegroundsHeroPickingViewModel vm) { ... }`. Casting DataContext can easily fail.